packages/cli/src/WorkflowRunner.ts

```diff
@@ -187,42 +188,44 @@ export class WorkflowRunner {
 			);
 		} else if (executionsProcess === 'main') {
 			executionId = await this.runMainProcess(data, loadStaticData, executionId, responsePromise);
+			void Container.get(InternalHooks).onWorkflowBeforeExecute(executionId, data);
```

Comment:
Instead of duplicating this code here, wouldn't it make sense to add those calls to `WorkflowExecuteAdditionalData` to the appropriate hook function on a hook that happens on `workflowExecuteBefore`?

Reply:
Currently the workflowExecuteBefore hook does not receive the required data necessary to invoke the telemetry functions